### PR TITLE
Minor type hints addition + version numbers tuning

### PR DESCRIPTION
**Pull request by pmderodat**

### CODE REVIEW

File: langkit/compile_context.py [hunk 1]
```diff
@@ -405,12 +408,21 @@ def __init__(self,
 
         :param lkt_semantic_checks: Whether to force Lkt semantic checks (by
             default, enabled only if ``types_from_lkt`` is true).
+
+        :param str version: String for the version of the generated library.
```
Comment (raph-amiard) [inline langkit/compile_context.py:412]: Remove the type annotations  here

File: langkit/dsl_unparse.py [hunk 2]
```diff
@@ -98,7 +98,7 @@ def class_from_location(loc):
 
         unit = DSLWalker.ctx.get_from_file(loc.file)
 
-        class_def = unit.root.lookup(lpl.Sloc(loc.line, 1))
+        class_def = unit.root.lookup(lpl.Sloc(loc.line, 99))
```
Comment (raph-amiard) [inline langkit/dsl_unparse.py:101]: lol

Comment (pmderodat) [reply langkit/dsl_unparse.py:101]: Indeed.

File: langkit/compile_context.py (hunk 1 above)
Comment (pmderodat) [reply langkit/compile_context.py]: Ah yes, thanks! Fixed now.

**Review by raph-amiard — APPROVED** [review]

**Pull request merged**